tools/scripts/redirect-gen.ts

```diff
@@ -0,0 +1,33 @@
+import fs from 'fs';
+import path from 'path';
+
+/* This can be used to create NGINX redirects, after running this function somewhere
+   build-curriculum.ts, it should be generated and visible in the file tree.
+*/
```

Comment:
```suggestion
/* This can be used to create NGINX maps for redirects. After running this
   script with `npx ts-node redirect-gen`, the map should appear in the current
   directory.
*/
```